overrides/overrides.go

```diff
@@ -0,0 +1,66 @@
+// Package overrides handles managing overrides in the running application
+package overrides
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/Masterminds/glide/msg"
+	gpath "github.com/Masterminds/glide/path"
+)
+
+var overrides map[string]*override
```

Comment:
I haven't seen any, but asking just in case - is there any possibility of parallel goroutine access to this map?